client/src/main/scala/skuber/api/client/impl/KubernetesClientImpl.scala

```diff
@@ -716,7 +717,8 @@ object KubernetesClientImpl {
 
     val requestMaker = (uri: Uri, method: HttpMethod) => HttpRequest(method = method, uri = uri)
 
-    val defaultClientSettings = ConnectionPoolSettings(actorSystem.settings.config)
+    val defaultClientConfigRoot = skuberConfig.getSkuberConfig("akka.http.host-connection-pool", _ => Some(skuberConfig.getRootSkuberConfig), actorSystem.settings.config)
```

Comment:
I think its better to avoid such hidden configurations, I suggest:
```
def apply(k8sContext: Context, logConfig: LoggingConfig, closeHook: Option[() => Unit], appConfig: Config, connectionPoolSettings: Option[ConnectionPoolSettings] = None)
```

```
val defaultClientConfigRoot = connectionPoolSettings.getOrElse(ConnectionPoolSettings(actorSystem.settings.config))
```

Reply:
wdyt?

Reply:
For me it is fine - it will be easier to provide some settings programmatically after applying your suggestion. I changed it that way. Can you take a look again?